docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+
+- Make sure you [start the event loop](#mainloop).
+
+- Implement your own widget class by inheriting the [Dali::Widget class](#widget-api).
+
+
+**Figure: DALi widget application**
+
+![DALi widget application](./media/dali_widget_application.png)
+
+For the steps of creating the basic parts of a simple DALi widget application, see [Basics of Creating a DALi Widget Application](#create).
+
+<a name="widget-application-api"></a>
+## Widget application class
+
+If you want to use DALi for your widget application, you must create the application with the `Dali::WidgetApplication` class. This class provides the means for initializing the resources required by DALi.
+
+The following table lists the main function you need in your widget application.
+
+**Table: Main functions**
+
+| Function                             | Description                              |
+|--------------------------------------|------------------------------------------|
+| `mainloop()`                         |
+| `RegisterWidgetCreatingFunction()`   | Register function to create widget instance |
+
+The `mainloop()` function is used to start the event loop. If you do not call the function and start the event loop, DALi cannot call any callback functions for application events.
+
+The `RegisterWidgetCreatingFunction()` function registers creating function to create widget instance. Widget application will create widget instance using this creating function when WidgetViewer application requests.
+
+<a name="widget-api"></a>
+## Widget class
+
+`Dali::Widget` class provides interface for creating custom widget.
+
+The following table lists the main function to manage widget instance.
+
+**Table: Main functions**
+
+| Function           | Description                                                                   |
+|--------------------|-------------------------------------------------------------------------------|
+| `OnCreate()`       | Called after the widget instance is created.                                  |
+| `OnTerminate()`    | Called after the widget instance is terminated.                               |
+| `OnPause()`        | Called when the widget is invisible.                                          |
+| `OnResume()`       | Called when the widget is visible.                                            |
+| `OnResize()`       | Called before the widget size is changed.                                     |
+| `OnUpdate()`       | Called when an event for updating the widget is received.                     |
+| `SetContentInfo()` | Send data about current state of widget instance to WidgetViewer application. |
```

Comment:
Send data about current state of widget instance to WidgetViewer application.

Change this sentence as:

This function sends data about current state of widget instance to widget viewer application.